### DISCUSSION TITLE
polyline in cytoscape.js

### DISCUSSION THREAD

**Discussion by Suneetha-Uppu** [cytoscape/cytoscape.js#2939] [Q&A]
I have a xml file in which the coordinates are specified. 

such as, 

graphics name ="CMBL, JS-1" fgcolor="#DA8E82" bgcolor="#FFFFFF" type="line" coords="1553,2469,1569,2469,1569,2469,1572,2469,1576,2468,1579,2466,1581,2464,1584,2461,1586,2459,1588,2456,1589,2452,1589,2449,1589,2449,1589,2324"

Using the coordinates, I want to draw straight lines(edges). May be in the above case, the position of source node is at "1553,2469" and target node is at "1589,2324". Rest other points are intermediate points. 

Can anyone please help me?

Thank you in advance.

**Comment by mike-kelly** [comment]
This should be achievable with the cytoscape-edge-editing extension.
